### DISCUSSION TITLE
GitHub

### DISCUSSION THREAD

**Discussion by romanovs1988** [community/community#137817] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

GitHub is cool!

**Comment by LiteBrite82** [comment]
👋 Welcome to the GitHub Community on Discussions, @romanovs1988 ! Happy to have you here 🙂

To get started, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776).
